python/mxnet/optimizer.py

```diff
@@ -433,7 +433,8 @@ def _get_wd(self, index):
 class SGD(Optimizer):
     """The SGD optimizer with momentum and weight decay.
 
-    The optimizer updates the weight by::
+    If any storage type of weight, state or grad is ``default``, \
+    **standard updates** are applied by::
```

Comment:
I think we should reverse the order and mention `lazy_update`, since the users of optimizer don't know how state stypes are created: 

```
If the storage types of weight and grad are both ``row_sparse``, and ``lazy_update`` is True,  **lazy updates** are applied by::
 
         for row in grad.indices:
            ...

Otherwise, **standard updates** are applied by:
          ...
```